MapboxVisionTests/RecordCoordinatorTests.swift

```diff
@@ -72,6 +70,69 @@ final class RecordCoordinatorTests: XCTestCase {
         XCTAssert(directoryExists(at: recordingPath), "Recording should be saved at \(recordingPath) after recording is stopped")
     }
 
+    class TestClipRequest {
+        let start: Float
+        let end: Float
+
+        init(start: Float, end: Float) {
+            self.start = start
+            self.end = end
+        }
+    }
+
+    func testVideoClipping(withRequests requests: [TestClipRequest], _ message: String = "", line: UInt = #line) {
+        let expectation = XCTestExpectation(description: "Wait for a big enough amount of time for record processing queue to finish its job")
+        let videoSource = FakeVideoSource()
+        videoSource.add(observer: coordinator)
+        videoSource.start()
+        coordinator.startRecording(referenceTime: 0, videoSettings: videoSettings) {}
+        _ = Timer.scheduledTimer(withTimeInterval: 4.0, repeats: false) { _ in
+            print("Make \(requests.count) clip\(requests.count == 1 ? "" : "s")")
```

Comment:
Please remove all prints from the function.

Reply:
done